### PR TITLE
[delivers #160937509] Fix schema bugs

### PR DESCRIPTION
**Pull request by ebsmith1515**

### CODE REVIEW

File: resources/schema/spidacalc/results/component.schema [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "description": "A component is a combination of an individual item and analysis type i.e. Pole Buckling",
   "type": "object",
   "required": [
-    "id", "poles", "actual", "ratio", "allowable", "failingRatio", "status", "resultsType", "analysisType"
+    "id", "actual", "ratio", "allowable", "failingRatio", "status", "resultsType", "analysisType"
```
Comment (ebsmith1515) [inline resources/schema/spidacalc/results/component.schema]: wire tension components do not have "poles"

unless I am missing something, we should not have gone this long before realizing that this was invalid against our schema. Missing tests?

File: src/main/groovy/com/spidasoftware/schema/conversion/changeset/v6/RevertBundleChangeset.groovy [hunk 2]
```diff
@@ -31,7 +31,7 @@ class RevertBundleChangeset extends AbstractCalcDesignChangeset {
                            "diameter",
                            "source"]
 
-        designJSON.structure.wires.each { Map wire ->
+        designJSON.structure?.wires.each { Map wire ->
```
Comment (spidadeveloper) [inline src/main/groovy/com/spidasoftware/schema/conversion/changeset/v6/RevertBundleChangeset.groovy]: add test for empty structure. check other changesets

Comment (spidadeveloper) [reply src/main/groovy/com/spidasoftware/schema/conversion/changeset/v6/RevertBundleChangeset.groovy]: Find a calc test with wire tension results that validates with schema
